docs/about/faq.md

```diff
@@ -122,6 +122,10 @@ the demand separately, because this needs some lead time for the IT department.
 
 Please refer to the [Venue page](../venue.md#parking) for parking information.
 
+## Call for papers
+
+Please deliver your presentations by July 10th latest.
+
```

Comment:
Shouldn't we add this to the Call for Papers page (https://boschglobal.github.io/embedded-linux/about/call_for_papers/)?

Reply:
additionally or instead of FAQ: I´m fine with both options

Reply:
Then I propose: instead of the FAQ.

Let's add the presentation submission deadline to the [Call for Papers page](https://boschglobal.github.io/embedded-linux/about/call_for_papers/).

Reply:
Oki Doki